.gitignore

```diff
@@ -35,3 +35,6 @@ projects/
 
 # Python requirements source directory
 src/
+
+# Python virtual environment
+venv/
```

Comment:
I think a more common name for the venv directory is `.venv`.

Reply:
I wasn't sure but now switched to `.venv`. Thanks for the review @alphan.